### ISSUE TITLE
Infinite critical error loop

### ISSUE THREAD

**Issue by RavanH** [duniter/sakia#711]
After fresh install and fist launch of Sakia, I get the first connexion screen. I fill out my account details and try to connect. Then it turns out that the dialog was about creating a new account, not connecting to an existing one (which is not clear at all) because I get a message about the account credentials already existing on the network.

After that, I get a critical error dialog with these details:

    Creating a client session outside of coroutine
    client_session: <aiohttp.client.ClientSession object at 0x7f4d2870ec88>

Problem is that when closing the error dialog with OK, a new one just opens... I cannot get out of this error loop.

**Comment by RavanH** [comment]
It even opens two or three new critical error dialogs at the same time... Some dialog details show different error details although difficult to copy them as a new error dialog opening prevents that.

One such error detail:
` 
Exception in callback Task._wakeup(<Future finished result=None>)
handle: <Handle Task._wakeup(<Future finished result=None>)>

    ----
    Traceback (most recent call last):

  File "asyncio/events.py", line 125, in _run

  File "asyncio/tasks.py", line 307, in _wakeup

  File "asyncio/tasks.py", line 291, in _step

KeyError: <QSelectorEventLoop running=False closed=False debug=False>
`

Another:
`
An unhandled exception occured

    ----
    Traceback (most recent call last):

  File "sakia/gui/navigation/network/table_model.py", line 90, in data

  File "sakia/data/processors/blockchain.py", line 51, in adjusted_ts

  File "sakia/data/processors/blockchain.py", line 88, in parameters

AttributeError: 'NoneType' object has no attribute 'parameters'
`

**Comment by Insoleet** [comment]
Which version of sakia did you use ?

**Comment by RavanH** [comment]
Hi @Insoleet that's version 0.31.6 

Starting from command line to test again, I also noticed that Sakia seems to think I have a QWERTY keyboard while I'm using AZERTY. Typing my passphrase will be impossible so I did not even try.

So not logging in (just closing the first connexion window) I get into the same error window loop. 

Terminal says:

`
ERROR:root:An unhandled exception occured
Traceback (most recent call last):
  File "sakia/gui/navigation/network/table_model.py", line 90, in data
  File "sakia/data/processors/blockchain.py", line 51, in adjusted_ts
  File "sakia/data/processors/blockchain.py", line 88, in parameters
AttributeError: 'NoneType' object has no attribute 'parameters'
ERROR:root:Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x7ff56029b748>
ERROR:root:Creating a client session outside of coroutine
client_session: <aiohttp.client.ClientSession object at 0x7ff56029b748>
ERROR:root:An unhandled exception occured
Traceback (most recent call last):
  File "sakia/gui/navigation/network/table_model.py", line 90, in data
  File "sakia/data/processors/blockchain.py", line 51, in adjusted_ts
  File "sakia/data/processors/blockchain.py", line 88, in parameters
AttributeError: 'NoneType' object has no attribute 'parameters'
ERROR:root:An unhandled exception occured
Traceback (most recent call last):
  File "sakia/gui/navigation/network/table_model.py", line 90, in data
  File "sakia/data/processors/blockchain.py", line 51, in adjusted_ts
  File "sakia/data/processors/blockchain.py", line 88, in parameters
AttributeError: 'NoneType' object has no attribute 'parameters'
ERROR:root:An unhandled exception occured
Traceback (most recent call last):
  File "sakia/gui/navigation/network/table_model.py", line 90, in data
  File "sakia/data/processors/blockchain.py", line 51, in adjusted_ts
  File "sakia/data/processors/blockchain.py", line 88, in parameters
AttributeError: 'NoneType' object has no attribute 'parameters'
ERROR:root:Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x7ff56029b748>
ERROR:root:Creating a client session outside of coroutine
client_session: <aiohttp.client.ClientSession object at 0x7ff56029b748>
^[ERROR:root:Unclosed response
client_response: <ClientResponse(ws://raspi3.cgeek.fr:80/ws/block) [101 Switching Protocols]>
<CIMultiDictProxy('Server': 'nginx', 'Date': 'Mon, 19 Jun 2017 20:42:31 GMT', 'Content-Type': 'application/octet-stream', 'Connection': 'upgrade', 'X-Sso-Wat': "You've just been SSOed", 'Set-Cookie': 'SSOwAuthRedirect=;; Path=/yunohost/sso/; Max-Age=Thu, Jan 01 1970 00:00:00 UTC;', 'Upgrade': 'websocket', 'Sec-Websocket-Accept': 'N2gZOapdf2sVbYW93TQu8YoHfqI=')>

ERROR:root:Unclosed response
client_response: <ClientResponse(wss://g1.duniter.org:443/ws/block) [101 Switching Protocols]>
<CIMultiDictProxy('Server': 'nginx/1.6.2', 'Date': 'Mon, 19 Jun 2017 20:42:55 GMT', 'Connection': 'upgrade', 'Upgrade': 'websocket', 'Sec-Websocket-Accept': 'UYE2DFcNs722xr+tlRWnNogwOME=')>

ERROR:root:Unclosed response
client_response: <ClientResponse(wss://g1.duniter.org:443/ws/block) [101 Switching Protocols]>
<CIMultiDictProxy('Server': 'nginx/1.6.2', 'Date': 'Mon, 19 Jun 2017 20:38:25 GMT', 'Connection': 'upgrade', 'Upgrade': 'websocket', 'Sec-Websocket-Accept': '6uNhqCBLmlUVNpCYdLVjdZIis3U=')>

ERROR:root:Unclosed connection
client_connection: Connection<('raspi3.cgeek.fr', 80, False)>
ERROR:root:Unclosed response
client_response: <ClientResponse(ws://raspi3.cgeek.fr:80/ws/block) [101 Switching Protocols]>
<CIMultiDictProxy('Server': 'nginx', 'Date': 'Mon, 19 Jun 2017 20:40:35 GMT', 'Content-Type': 'application/octet-stream', 'Connection': 'upgrade', 'X-Sso-Wat': "You've just been SSOed", 'Set-Cookie': 'SSOwAuthRedirect=;; Path=/yunohost/sso/; Max-Age=Thu, Jan 01 1970 00:00:00 UTC;', 'Upgrade': 'websocket', 'Sec-Websocket-Accept': '+k8FIkAl+9X3S2ARpj5UrO0EHSM=')>

ERROR:root:Unclosed connection
client_connection: Connection<('g1.duniter.d0p1.eu', 80, False)>
ERROR:root:Unclosed response
client_response: <ClientResponse(ws://g1.duniter.d0p1.eu:80/ws/block) [101 Switching Protocols]>
<CIMultiDictProxy('Server': 'nginx/1.12.0', 'Date': 'Mon, 19 Jun 2017 20:41:01 GMT', 'Connection': 'upgrade', 'Upgrade': 'websocket', 'Sec-Websocket-Accept': 'c6OB5JbMZllRa9TEwq7OEw2N7BA=')>

ERROR:root:Unclosed response
client_response: <ClientResponse(ws://g1.duniter.d0p1.eu:80/ws/block) [101 Switching Protocols]>
<CIMultiDictProxy('Server': 'nginx/1.12.0', 'Date': 'Mon, 19 Jun 2017 20:43:16 GMT', 'Connection': 'upgrade', 'Upgrade': 'websocket', 'Sec-Websocket-Accept': '3dUqETmG0bjs2T79CufOP5Dbucs=')>
`

This is on Ubuntu GNOME 17.04 running a Wayland session.
